remedy.lic

```diff
@@ -284,67 +268,25 @@ class Remedy
       waitrt?
       stow_item(left_hand)
       get_item('sieve')
-#      mix("push my #{@noun} with my sieve")
       command = "push my #{@noun} with my sieve"
     when 'Once finished you notice clumps of material'
       waitrt?
-#      mix("turn my #{@container} with my mixing stick")
       command = "turn my #{@container} with my mixing stick"
     when 'Patiently you wait a few moments and then proceed'
       waitrt?
-#      mix("mix my #{@container} with my mixing stick")
       command = "mix my #{@container} with my mixing stick"
     when 'Applying the final touches', 'Interesting thought really... but no.', 'Try as you might'
       finish
     when 'Roundtime:'
       waitrt?
       command = "mix my #{@container} with my mixing stick"
-    when 'To avoid a problem'
-      case bput("analyze my #{@noun}",
-                'appears free of defects that would impede further preparation.',
-                'composition looks accurate and ready for continued preparation',
-                'smell the mixture',
-                'You need to be holding the mixing stick first.',
-                'You do not see anything that would prevent preparing',
-                'with a sieve',
-                'Clumps of material stick to the sides',
-                'Applying the final touches',
-                'Interesting thought really... but no.',
-                'easy piece to make',
-                'basic piece to make',
-                'simple piece to make',
-                'You try to analyze',
-                'You filter',
-                'Try as you might',
-                'You spin')
-      when 'You do not see anything that would prevent preparing', 'appears free of defects that would impede further preparation.', 'composition looks accurate and ready for continued preparation', 'You try to analyze', 'You filter', 'You spin'
-        stow_item(left_hand)
-        get_item('mixing stick')
-#        mix("mix my #{@container} with my mixing stick")
-        command = "mix my #{@container} with my mixing stick"
-      when 'smell the mixture'
-        command = "smell my #{@noun}"
-      when 'You need to be holding the mixing stick first'
-        stow_item(left_hand)
-        get_item('mixing stick')
-#        mix("mix my #{@container} with my mixing stick")
-        command = "mix my #{@container} with my mixing stick"
-      when 'Clumps of material stick to the sides'
-        waitrt?
-        command = "turn my #{@container} with my mixing stick"
-      when 'Applying the final touches', 'Interesting thought really... but no.', 'easy piece to make', 'basic piece to make', 'simple piece to make', 'Try as you might'
-        finish
-      when 'with a sieve'
-        stow_item(left_hand)
-        get_item('sieve')
-        command = "push my #{@noun} with my sieve"
-      end
+    when 'To avoid a problem', 'That tool does not seem suitable for that task.'
+      analyze_mix     
```

Comment:
Oh, huh... I bet the call to `mix` inside `analyze_mix` will actually end up calling `finish`, which exits the script before control ever returns here. Neat, if a bit unintuitive.